### ISSUE TITLE
compilation breaks on "Lint found errors in the project; aborting build."

### ISSUE THREAD

**Issue by hvdwolf** [lbdroid/HFPClient#1]
Upon compilation I got an error
```
:app:assembleRelease
:app:assemble
:app:lint
Ran lint on variant release: 359 issues found
Ran lint on variant debug: 359 issues found
Wrote HTML report to file:///media/harryvanderwolf/64GB/software/lbdroid/HFPClient/app/build/reports/lint-results.html
Wrote XML report to file:///media/harryvanderwolf/64GB/software/lbdroid/HFPClient/app/build/reports/lint-results.xml
:app:lint FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lint'.
> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 11.352 secs

```

So I added the:
```
    lintOptions {
        abortOnError false
    }
```
to the HFPClient/app/build.gradle

Now it compiles fine.
I have not tested the apk yet

**Edit**: This is on a 64bit Ubuntu derived OS.
```
uname -a
Linux Swanky 4.8.17-galliumos #1 SMP PREEMPT galliumos4 Thu Feb 23 02:27:28 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by lbdroid** [comment]
Fancy that yours did a lint check but I had to force mine to do it.
I took a quick peek at the lint errors, and it looks like the majority of them are because I've temporarily disabled some features that were present in the qualcomm repo. Eventually it would be nice to actually bring those features back, so I agree that it is probably best to just ignore them for now rather than trying to fix the errors.

I can add the change myself if you like, but if you'd like a more obvious accreditation, I'm always open to pull requests.
